CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fix for `AbstactRestClient` failing to return when streaming a larger file [#1805](https://github.com/zowe/zowe-cli/issues/1805)
```

Comment:
Since 3 issues will be resolved, should we list all 3?

Reply:
I've updated the CHANGELOG to include all three issues.